src/frontend/src/lib.rs

```diff
@@ -14,6 +14,8 @@
 
 #![feature(assert_matches)]
 
+pub type AnyMap2 = anymap::Map<dyn core::any::Any + Send + Sync>;
```

Comment:
There's actually a public crate named `anymap2`. This type alias here is confusing.

Reply:
Any suggestion for naming this alias?

Reply:
How about just `Plugins`?